presto-main/src/main/java/io/prestosql/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -820,14 +820,14 @@ private Void visitScanFilterAndProjectInfo(
         private String printDynamicFilters(Collection<DynamicFilters.Descriptor> filters)
         {
             return filters.stream()
-                    .map(filter -> filter.getId() + " -> " + filter.getInput())
+                    .map(filter -> filter.getInput() + " " + filter.getOperator().getValue() + " #" + filter.getId())
```

Comment:
Could you paste an example of what the explain output looks like after this change ?

Reply:
```sql
EXPLAIN SELECT l.comment FROM  lineitem l, part p WHERE p.partkey > l.partkey AND p.comment = 'onic deposits'
```
results in:
```
Fragment 0 [SINGLE]
    Output layout: [comment]
    Output partitioning: SINGLE []
    Stage Execution Strategy: UNGROUPED_EXECUTION
    Output[comment]
    │   Layout: [comment:varchar(44)]
    │   Estimates: {rows: ? (?), cpu: ?, memory: ?, network: ?}
    └─ RemoteSource[1]
           Layout: [comment:varchar(44)]

Fragment 1 [SOURCE]
    Output layout: [comment]

    Output partitioning: SINGLE []
    Stage Execution Strategy: UNGROUPED_EXECUTION
    FilterProject[filterPredicate = ("partkey" < "partkey_0")]
    │   Layout: [comment:varchar(44)]
    │   Estimates: {rows: ? (?), cpu: ?, memory: ?, network: ?}/{rows: ? (?), cpu: ?, memory: ?, network: ?}
    └─ CrossJoin
       │   Layout: [partkey:bigint, comment:varchar(44), partkey_0:bigint]
       │   Estimates: {rows: ? (?), cpu: ?, memory: ?, network: ?}
       │   Distribution: REPLICATED
       │   dynamicFilterAssignments = {partkey_0 -> #df_350}
       ├─ ScanFilter[table = memory:2, grouped = false, filterPredicate = true, dynamicFilter = {"partkey" < #df_350}]
       │      Layout: [partkey:bigint, comment:varchar(44)]
       │      Estimates: {rows: ? (?), cpu: ?, memory: 0B, network: 0B}/{rows: ? (?), cpu: ?, memory: 0B, network: 0B}
       │      partkey := 1
       │      comment := 15
       └─ LocalExchange[SINGLE] ()
          │   Layout: [partkey_0:bigint]
          │   Estimates: {rows: ? (?), cpu: ?, memory: 0B, network: ?}
          └─ RemoteSource[2]
                 Layout: [partkey_0:bigint]

Fragment 2 [SOURCE]
    Output layout: [partkey_0]
    Output partitioning: BROADCAST []
    Stage Execution Strategy: UNGROUPED_EXECUTION
    ScanFilterProject[table = memory:3, grouped = false, filterPredicate = ("comment_1" = CAST('onic deposits' AS varchar(23)))]
        Layout: [partkey_0:bigint]
        Estimates: {rows: ? (?), cpu: ?, memory: 0B, network: 0B}/{rows: ? (?), cpu: ?, memory: 0B, network: 0B}/{rows: ? (?), cpu: ?, memory: 0B, network: 0B}
        comment_1 := 8
        partkey_0 := 0
```

Reply:
this would require doc update

Reply:
Updated `presto-docs/src/main/sphinx/admin/dynamic-filtering.rst`.